src/gui/qgsattributedialog.h

```diff
@@ -95,6 +95,14 @@ class GUI_EXPORT QgsAttributeDialog : public QDialog
      */
     bool event( QEvent *e ) override;
 
+    /**
+     * Sets an additional expression context scope to be used
+     * for calculations in this form.
```

Comment:
Is it an "additional"/"extra" scope or is it just "the" scope used in the expression context for the calculations?
Anyway, maybe add the param like: `Sets \a extraScope as the additional expression...`

Reply:
Other scopes are already there (global, project, layer, form) and this one can be provided from the exterior (e.g. map tool capture)
I'll adjust